docs/content/en/docs/getting-started/optional/irsa.md

```diff
@@ -115,70 +114,75 @@ You must use a single OIDC provider per EKS Anywhere cluster, which is the best
         serviceAccountIssuer: https://$ISSUER_HOSTPATH
     ```
 
-Set the remaining fields in cluster spec as required and create the cluster using the `eksctl anywhere create cluster` command. 
+Set the remaining fields in cluster spec as required and create the cluster.
 
 #### Generate keys.json and make it publicly accessible
 
-1. The cluster provisioning workflow generates a pair of service account signing keys. Retrieve the public signing key generated and used by the cluster, and create a keys.json document containing the public signing key.
+1. The cluster provisioning workflow generates a pair of service account signing keys. Retrieve the public signing key from the cluster and create a keys.json document with the content.
 
     ```bash
     git clone https://github.com/aws/amazon-eks-pod-identity-webhook
     cd amazon-eks-pod-identity-webhook
-    kubectl get secret ${CLUSTER_NAME}-sa -n eksa-system -o jsonpath={.data.tls\\.crt} | base64 --decode > ${CLUSTER_NAME}-sa.pub    
+    kubectl get secret ${CLUSTER_NAME}-sa -n eksa-system -o jsonpath={.data.tls\\.crt} | base64 --decode > ${CLUSTER_NAME}-sa.pub
     go run ./hack/self-hosted/main.go -key ${CLUSTER_NAME}-sa.pub | jq '.keys += [.keys[0]] | .keys[1].kid = ""' > keys.json
     ```
-1. Upload the keys.json document to the s3 bucket.
+
+1. Upload the keys.json document to the S3 bucket.
     ```bash
     aws s3 cp --acl public-read ./keys.json s3://$S3_BUCKET/keys.json
     ```
 
 ### Deploy pod identity webhook
 
-After hosting the service account public signing key and OIDC discovery documents, the applications running in pods can start accessing the desired AWS resources, as long as the pod is mounted with the right service account tokens. This part of configuring the pods with the right service account tokens and env vars is automated by the [Amazon Pod Identity Webhook](https://github.com/aws/amazon-eks-pod-identity-webhook). Once the webhook is deployed, it mutates any pods launched using service accounts annotated with `eks.amazonaws.com/role-arn`. Follow the steps below to deploy the Amazon EKS Pod Identity Webhook to your cluster:
+The [Amazon Pod Identity Webhook](https://github.com/aws/amazon-eks-pod-identity-webhook) configures pods with the necessary environment variables and file mounted tokens to interact with AWS services. The webhook will configure any pod associated with a service account that has an `eks-amazonaws.com/role-arn` annotation.
```

Comment:
The first sentence sounds a little weird. Do you mean "...and file-mounted tokens to interact..."?

Reply:
Adjusted to be clearer.